Gatsby/src/pages/catalog.js

```diff
@@ -8,6 +9,7 @@ import Features from "../components/Catalog/Features/features"
 
 const MembersPage = () => {
   const [filter, setFilter] = useState([]);
+  const [categories] = useState(DataFile)
```

Comment:
@kayceeDev It would be better to use another naming here instead of categories

Reply:
What do you suggest?

Reply:
"dataFile" can be a good option

Reply:
@kayceeDev

Reply:
ok. I will send a PR now